src/views/Correction/CorpCorrection.vue

```diff
@@ -78,7 +78,7 @@ import { BcCorrectionResource, BenCorrectionResource, CccCorrectionResource, Ulc
     CompletingParty
   }
 })
-export default class BcCorrection extends Mixins(CommonMixin, DateMixin, FeeMixin, FilingTemplateMixin) {
```

Comment:
Renamed so it's not mistaken for a page specific to entity type "BC" (BC Corporation).